### ISSUE TITLE
Sanity check for Build OSCAR Client Image

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#124]
**Reported by bli on 22 Jul 2005 20:08 UTC**
```
A quick check for the existence of the files for:

Package File, Packages Directory, Disk Partition File

should be made PRIOR to the image being built.
```

**Comment by dikim33** [comment]
**Comment by efocht on 4 Aug 2006 08:51 UTC**
Check for existence of rpmlists file and disktable file is trivial, the repository check is more complex, and probably hard to do if the repository is online. I'd fix the easy things first and leave the repository check for later.

**Comment by dikim33** [comment]
**Comment by anonymous on 4 Aug 2006 09:11 UTC**
error: %pre(nfs-utils-1.0.6-46.i386) scriptlet failed, exit status 255
error:   install: %pre scriptlet failed (2), skipping nfs-utils-1.0.6-46
error: %pre(postfix-2.1.5-2.3.RHEL4.1.i386) scriptlet failed, exit status 255
error:   install: %pre scriptlet failed (2), skipping postfix-2.1.5-2.3.RHEL4.1
error: %pre(xorg-x11-xdm-6.8.1-23.EL.i386) scriptlet failed, exit status 255
error:   install: %pre scriptlet failed (2), skipping xorg-x11-xdm-6.8.1-23.EL
256: xorg-x11-tools-6.8.1-23.EL
error: %pre(sendmail-8.13.1-2.i386) scriptlet failed, exit status 255
error:   install: %pre scriptlet failed (2), skipping sendmail-8.13.1-2
257: kbd-1.12-2
Rpm installation failed. at /usr/lib/systeminstaller/SystemInstaller/Package.pm line 217
Failed to install files. at /usr/bin/mksiimage line 184
258: 2006-7-4 14:19:29 [:: Line 185](main) Unmounting /proc.
259: 2006-7-4 14:19:29 [:: Line 68](SystemInstaller::Image) si_rmimage oscarimageFATAL: Can't remove /var/lib/systemimager/scripts/oscarimage.master!
Package install failed
 at /usr/bin/mksiimage line 190
 at /usr/bin/mksiimage line 191
Built image from rpms
Image oscarimage does not exist
 at /usr/bin/mksidisk line 85
Couldn't run command mksidisk -A --name oscarimage --file /opt/oscar/oscarsamples/scsi.disk at /usr/lib/perl5/vendor_perl/5.8.0/i386-linux-thread-multi/Tk.pm line 247
Use of uninitialized value in hash element at /usr/lib/perl5/vendor_perl/5.8.0/i386-linux-thread-multi/Tk/Trace.pm line 181.
Requested images not found.
 at /usr/bin/mksiimage line 292

**Comment by dikim33** [comment]
**Comment by bli on 4 Aug 2006 09:28 UTC**
What does the attachment/comment have to do with this ticket?  This looks like somebody forgot to disable SELinux.

If you have support questions, please post them to the oscar-users list:

https://lists.sourceforge.net/lists/listinfo/oscar-users

**Comment by dikim33** [comment]
**Comment by jparpail on 10 Apr 2007 12:43 UTC**
Not critical. Changing milestone to 5.1

**Comment by dikim33** [comment]
**Modified by jparpail on 11 Apr 2007 12:00 UTC**

**Comment by dikim33** [comment]
**Modified by efocht on 11 Apr 2007 12:24 UTC**

**Comment by dikim33** [comment]
**Modified by valleegr on 30 Jun 2008 06:48 UTC**

**Comment by dikim33** [comment]
**Modified by valleegr on 7 Feb 2009 18:40 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 8 Feb 2009 20:05 UTC**
(In [7986]) This check-in fixes #126 - Implement a sanity check before the actual creation of 
a new image.
